pyanaconda/ui/tui/spokes/user.py

```diff
@@ -130,11 +126,31 @@ def status(self):
         else:
             return _("User %s will be created") % self.data.user.userList[0].name
 
+    def input(self, args, key):
+        self.dialog.wrong_input_message = None
+        try:
+            field = self.visible_fields[int(key)-1]
+        except (ValueError, IndexError):
+            pass
+        else:
+            if field.attribute == "gecos":
+                self.dialog.wrong_input_message = _("Full name can't contain ':' character")
+            elif field.attribute == "name":
+                # more granular message is returned by check_name
+                pass
+            elif field.attribute == "_groups":
+                self.dialog.wrong_input_message = _("Either some group name is invalid or groups are not separated by comma")
```

Comment:
Again just a nitpick/suggestion:

"Either **a** group name **in the group list** is invalid or groups are not separated by **a** comma"

Sounds tiny bit better that way IMHO.